build/Targets/Settings.targets

```diff
@@ -19,6 +19,8 @@
 
     <!-- Turn on hard-linking on Windows to reduce copy time and disk space -->
     <CreateHardLinksForCopyLocalIfPossible Condition="'$(OS)' == 'Windows_NT'">true</CreateHardLinksForCopyLocalIfPossible>
+
+    <ManagedExecutor Condition="'$(OS)' != 'Windows_NT'">mono</ManagedExecutor>
```

Comment:
Needs to be `$(MonoToolsetPath)` to account for https://github.com/dotnet/roslyn/commit/34870a4d27547b9ba628cbbc4a02028648d62d82#diff-e9ce2aa940206f8c6f45f33259fb5c09R38